recipes/patchelf/patchelf.inc

```diff
@@ -0,0 +1,12 @@
+SRC_URI = "http://nixos.org/releases/${PN}/${P}/${P}.tar.bz2"
+LICENSE = "GPLv3"
+SUMMARY = "Tool to allow editing of RPATH and interpreter fields in ELF binaries"
+
+inherit autotools c++ c
+
+RECIPE_TYPES = "native canadian-cross"
+
+DEP = "host:libc host:libgcc host:libm host:libstdc++"
+RDEPENDS_${PN} += "${DEP}"
+DEPENDS_${PN} += "${DEP}"
+EXTRA_OECONF += "CFLAGS=-static CXXFLAGS=-static"
```

Comment:
Using it to patch the sdk binaries in an sdk-install script. Having it depend on the sdk provided libc/stdc++ will require a wrapper script for patchelf only. The benefit would be saving <1 MB (32 bit arm). But ads complexity I do not want.